test/library/packages/UnitTest/UnitTest.chpl

```diff
@@ -9,9 +9,15 @@ Any function of the form
 is treated as a test function. These functions must accept an object of Test
 Class. We use :proc:`~UnitTest.runTest` to pass the tests.
 
-**Example**
+.. note::
 
-*Using Asserts*
+  Tests must be launched with a test launcher. Currently, this is a standalone application 
+  built separately. However, this functionality will eventually be integrated into mason.
+
+Assert Functions
+================
+
+Here are the assert functions available in the UnitTest module:
 
 :proc:`~Test.skip`
```

Comment:
Can we put these in a list? e.g.

```
- :proc:`~Test.skip`
- ....
```

Reply:
I was also thinking of doing that. Thanks